test/acceptance/features/mutableFailingSBR.feature

```diff
@@ -0,0 +1,116 @@
+Feature: Unsuccessful Service Binding Resource should be mutable
+
+    As a user of Service Binding operator
+    I should be able to reapply my SBR since the previous SBR was unsuccessful
```

Comment:
if it was unsuccessful, we are not reapplying the binding, instead we update the binding. Can we reflect it in the above text?

Reply:
While we can mention update, yes, in the text, that would map to a kubectl apply action right? 
For example, I am envisioning a scenario wherein I applied a yaml and my Status ready is false. Then I am reapplying as a user.(not waiting on reconcile as reconcile is triggered on create and delete).

Reply:
ok